sources/packages/backend/env_setup_apps.js

```diff
@@ -0,0 +1,11 @@
+const path = require("path");
+if (
+  process.env.NODE_ENV !== "production" ||
+  process.env.NODE_ENV !== "docker"
+) {
+  require("dotenv").config({
+    path: path.resolve(__dirname, "../../../../../../.env"),
```

Comment:
just curios not a blocker, is there a way to avoid `../../../../../` I saw in it required("../../../../../../") too

Reply:
Not that I know. I did not spend much time on this. To be honest I would like to get rid of this completely.
For instance, we have the below tech-debit to explore out-of-box Nestjs features that are not used right now:
https://app.zenhub.com/workspaces/student-information-management-system-5fce9df5aa1b45000e937014/issues/bcgov/sims/1099
One of the items is related to the env configs. Under the hood apparently, both will use the `dotenv` package, but at least we can get rid of the `require` js file.
Not a chance that I would be willing to do alongside the monorepo changes. Is it ok @ann-aot?